src/ralph/domains/admin.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+from django.utils.translation import ugettext_lazy as _
+
+from ralph.admin import RalphAdmin, RalphTabularInline, register
+from ralph.domains.models.domains import (
+    Domain,
+    DomainContract,
+    DomainRegistrant
+)
+
+
+class DomainContractInline(RalphTabularInline):
+    model = DomainContract
+    extra = 0
+
+
+@register(Domain)
+class DomainAdmin(RalphAdmin):
+    pass
+
+    list_select_related = ['technical_owner', 'business_owner', 'domain_holder']
+    list_filter = [
+        'name', 'service_env',
+    ]
+    list_display = [
+        'name', 'parent', 'business_owner',
+        'technical_owner', 'domain_holder'
+    ]
+    fieldsets = (
+        (_('Basic info'), {
+            'fields': (
+                'name', 'parent', 'remarks', 'domain_status',
+            )
+        }),
+        (_('Ownership info'), {
+            'fields': (
+                'service_env', 'domain_status',
+                'business_segment', 'business_owner',
+                'technical_owner', 'domain_holder'
+            )
+        })
+    )
+    search_fields = ['name', ]
+    inlines = (DomainContractInline, )
+
+
+@register(DomainContract)
+class DomainContract(RalphAdmin):
+    list_select_related = ['domain', 'domain__service_env']
+    list_filter = [
+        'domain__name', 'domain__service_env__service__name'
+    ]
+    list_display = [
+        'domain', 'expiration_date', 'price'
```

Comment:
what about service? if we allow to filter by service name shouldn't it be one of the list columns too?